styles.css

```diff
@@ -68,5 +68,50 @@ p {
   justify-content: center;
   margin-top: 2rem;
 }
+.buttons{
+  display: flex; 
+  flex-direction: row;
+  justify-content: flex-end;
+}
+.button {
+padding: .6rem;
+border: medium solid black; 
+margin-left: .5rem;
+margin-right: .5rem; 
+cursor: pointer;
+
+
+}
+.title1{
+  display: flex;
+  font-size: 3rem;
+  justify-content: center; 
+  align-items: center;
+  height: 100vh
+}
+
 
-  
+.vbcourt {
+  background-image: linear-gradient(rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.5)), url('your-image.jpg');
+  background-size: cover; 
+  background-position: center;
+  background-repeat: no-repeat; 
+}
+.dropdown{
+  position: relative;
+  display: inline-block;
+}
+#accountButton {
+padding: 16px;
+font-size: 16px;
+}
+.dropdown-content{
```

Comment:
the text in this dropdown isn't readable because of the yellow-on-white. I'd recommend changing it to a darker color or changing the background color in this style